packages/sourcecred/package.json

```diff
@@ -28,6 +30,7 @@
     "d3-shape": "^2.0.0",
     "d3-time": "^2.0.0",
     "d3-time-format": "^3.0.0",
+    "date-fns": "2.23.0",
```

Comment:
is this being used? I see you using `@date-io/date-fns` instead.

Reply:
installed it for handling the date formats in the filter, later on, will remove it if I don't use it

Reply:
the `@date-io/date-fns` library xposes an abstraction interface over date-fns for the provider to consume the actual functionality exists in date-fns